### ISSUE TITLE
"typeof" ignores round brackets

### ISSUE THREAD

**Issue by DPOH-VAR** [babel/babel#2771]
```javascript
typeof a(); // ==> typeof a();
(typeof a)(); // ==> typeof a();
```
Tested with Babel repl ``5.8.33``

**Comment by loganfsmyth** [comment]
This would appear to be fixed in Babel 6, this it still has it's own issues at the moment.

**Comment by thejameskyle** [comment]
@loganfsmyth What other issues?

**Comment by sebmck** [comment]
Fixed via cdeff2fbb1f55b3b15a495520415f1b4f0f0b5cb.
